tests/functional/centos/7/cluster/hosts

```diff
@@ -1,6 +1,6 @@
 [mons]
 mon0 monitor_address=192.168.1.10
-mon1 monitor_interface=eth1
+mon1 monitor_address=192.168.1.11
```

Comment:
`monitor_interface` needs to be set at least in group_vars/all.yml. That why I set `monitor_interface` [here](https://github.com/ceph/ceph-ansible/pull/1594/files#diff-c40942992a7c572549d5f50fe58bc997R7).
In CI testing case I don't see the point to set it both in all.yml and in inventory host file.